mappings/net/minecraft/block/MaterialColor.mapping

```diff
@@ -24,7 +30,8 @@ CLASS net/minecraft/class_1243 net/minecraft/block/MaterialColor
 	FIELD field_26260 BLACK_TERRACOTTA Lnet/minecraft/class_1243;
 	FIELD field_26261 color I
 	FIELD field_26262 id I
-	FIELD field_26264 AIR Lnet/minecraft/class_1243;
+	FIELD field_26263 BLOCK_COLORS [Lnet/minecraft/class_1243;
```

Comment:
`COLORS`

Reply:
^

Reply:
Thats already taken, this ones for all the 16 block color types..like all 16 concrete types

Reply:
https://github.com/DeadlyMC/Legacy-yarn/commit/10124cb7d6b92d654af16104cda01e6dcf5f28d2